src/thunderbots/software/ai/intent/dribble_intent.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "ai/intent/intent.h"
+#include "ai/primitive/dribble_primitive.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+
+class DribbleIntent : public Intent, public DribblePrimitive
+{
+   public:
+    static const std::string INTENT_NAME;
+    /**
+     * Creates a new Dribble Intent
+     * Moves the robot in a straight line between its current position and the given
```

Comment:
nit; should be an empty line between the short and long description here (ie.above this line)

Reply:
done